packages/react-core/src/components/SearchInput/SearchInput.tsx

```diff
@@ -298,6 +296,18 @@ const SearchInputBase: React.FunctionComponent<SearchInputProps> = ({
             getAttrValueMap={getAttrValueMap}
             isSearchMenuOpen={isSearchMenuOpen}
           />
+        </span>
+      );
+
+      return (
+        <div className={css(className)} ref={searchInputRef} {...props}>
+          <Popper
+            trigger={buildSearchTextInputGroupWithExtraButtons({ id: 'custom-advanced-search' })}
+            popper={advancedSearch}
+            isVisible={isSearchMenuOpen}
+            enableFlip={false}
+            appendTo={() => document.querySelector('#custom-advanced-search')}
```

Comment:
We cannot hardcode this value. It should be something that is configurable the prop could be the same or inspired by the appendTo prop from TimePicker.tsx
```
/** The container to append the advanced search form to. Defaults to 'parent'.
   * If your form is being cut off you can append it to an element higher up the DOM tree.
   * Some examples:
   * appendTo="parent"
   * appendTo={() => document.body}
   * appendTo={document.getElementById('target')}
   */
  appendTo?: HTMLElement | (() => HTMLElement) | 'inline' | 'parent';
```

Reply:
I am not sure about that. 
We want to append this menu to 'SearchTextInputGroup' (we set the value 'custom-advanced-search' as id) every time and not to an external DOM element. 

Maybe I am missing something.
Could you please provide an example of where we need to configure this element outside the component?

Reply:
It's possible that someone will put an advanced search into a Wizard modal for example and that would get clipped or formatted incorrectly relative to the wizard or modal. Those tend to be some the edge cases we are allowing for. 

Additionally, if someone wants an inline advanced search, where opening the modal does (for whatever reason) push down the contents of the page so the advanced search is not overlaying the results, that should be possible using the 'inline' value of 'appendTo'

Reply:
Got it, thanks! 👍🏼 
So I think we can:
- keep the current behavior as default (without passing the prop appendTo). (popper)
- provide the inline option to push down the content ( no popper)
- add the possibility to pass an external DOM element (popper)